webpack.common.js

```diff
@@ -62,7 +62,7 @@ module.exports = {
 				test: /\.(svg)$/i,
 				use: [
 					{
-						loader: 'url-loader'
+						loader: 'svg-inline-loader'
```

Comment:
@ChristophWurst, 
@skjnldsv proposed to import svg as string but it's configured with url-loader in the webpack config, can we move on with svg-inline-loader? Because otherwise we have to find with another way how to import it

Reply:
> url-loader works like file-loader, but can return a DataURL if the file is smaller than a byte limit.

so it already inlines in most cases. But fine by if we switch to the svg-inline-loader if that doesn't break anything else. IIRC the CKEditor needed some special loaders. Let's test that carefully so it doesn't break :)